### PR TITLE
Add support for multi-channel setups

### PR DESCRIPTION
**Pull request by AurelienGasser**
**TL;DR**: Make `appChannel` a list.

- @ClementGautier Please note that this is a [breaking change](https://github.com/SubstraFoundation/hlf-k8s/pull/75/files#diff-fcbde7c13ef662a06d6249e260a01a95R13)
- @thibaultrobert FYI

---

**Supports adding new channels on the fly:**

1. `skaffold run`

2. Edit skaffold.yaml

Add the following lines:

```yaml
# orderer
appChannels[1].channelName: yourchannel

# peer-1
appChannels[1].channelName: yourchannel
appChannels[1].organizations[0].org: MyOrg1
appChannels[1].organizations[0].mspid: MyOrg1MSP
appChannels[1].organizations[0].configUrl: network-org-1-peer-1-hlf-k8s-config-operator.org-1/config/configOrgWithAnchors.json
appChannels[1].organizations[1].org: MyOrg2
appChannels[1].organizations[1].mspid: MyOrg2MSP
appChannels[1].organizations[1].configUrl: network-org-2-peer-1-hlf-k8s-config-operator.org-2/config/configOrgWithAnchors.json
appChannels[1].policies: |
  Readers:
      Type: ImplicitMeta
      Rule: "ANY Readers"
  Writers:
      Type: ImplicitMeta
      Rule: "ANY Writers"
  Admins:
      Type: ImplicitMeta
      Rule: "ANY Admins"
appChannels[1].chaincodePolicy: OR('MyOrg1MSP.member'\,'MyOrg2MSP.member')
appChannels[1].chaincodeName: mycc
appChannels[1].chaincodeVersion: "1.0"

# peer-2
appChannels[1].channelName: yourchannel
appChannels[1].chaincodePolicy: OR('MyOrg1MSP.member'\,'MyOrg2MSP.member')
appChannels[1].chaincodeName: mycc
appChannels[1].chaincodeVersion: "1.0"
```

3. `skaffold run`

A new monitor pod was created

```
$ kubectl get pods -n orderer | grep monitor
network-orderer-hlf-k8s-monitor-mychannel-74b8f8fb46-kbc85        1/1     Running   0          23m
network-orderer-hlf-k8s-monitor-yourchannel-7b8d46c64f-p6sdc      1/1     Running   0          20m
```

The channel `yourchannel` was created and organizations were added to it:

```
$ kubectl logs -n orderer network-orderer-hlf-k8s-monitor-yourchannel-7b8d46c64f-p6sdc | tail -10
[
  "MyOrderer",
  "MyOrg1",
  "MyOrg2"
]
[DEBUG] Application channel: yourchannel
[
  "MyOrg1",
  "MyOrg2"
]
```

### CODE REVIEW

**Comment by AurelienGasser** [comment]
E2E tests: :heavy_check_mark:

File: charts/hlf-k8s/Chart.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@
 apiVersion: v1
 name: hlf-k8s
 home: https://substra.org/
-version: 1.4.6
+version: 1.5.0
```
Comment (ClementGautier) [inline charts/hlf-k8s/Chart.yaml:18]: if it's a breaking change (and it is), you should probably  bump to 2.0.0

File: charts/hlf-k8s/README.md [hunk 2]
```diff
@@ -10,6 +10,13 @@ Hlf-k8s runs Hyperledger Fabric v1.4.
 
 - Kubernetes 1.14+
 
+## Changelog
+
+### 1.5.0
+
+- `appChannel` changed to `appChannels` (list)
+- `appChannel.name` changed to `appChannels[].channelName`
+
```
Comment (ClementGautier) [inline charts/hlf-k8s/README.md:19]: Could you move that into a CHANGELOG.md file ? Also having an UPGRADE.md could be nice

Comment (AurelienGasser) [reply charts/hlf-k8s/README.md:19]: will do

**Comment by AurelienGasser** [comment]
@Kelvin-M @ClementGautier this is ready for review!

**Pull request merged**